tests/unit/components/message-chat-test.js

```diff
@@ -0,0 +1,22 @@
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent('message-chat', 'Unit | Component | message-chat', {
+  unit: true
+});
+
+test('#formattedContent turns URLs into links', function(assert) {
+  let component = this.subject({
+    message: { content: 'visit kosmos.org for more info' }
```

Comment:
This is not actually a URL being turned into a link, but something that resembles an FQDN. I think we should only link actual URLs, because we can't know if there's anything behind that domain name, much less if it's via HTTPS.